courses/models_test.py

```diff
@@ -244,3 +250,13 @@ def test_course_page_properties():
     assert course.video_title == "<p>title</p>"
     assert course.video_url == "http://test.com/mock.mp4"
     assert course.background_image.title == "background-image"
+    who_should_enroll_page = WhoShouldEnrollPageFactory(
```

Comment:
Do we not need to add tests for ProgramPage properties ?

Reply:
Done.